docs/DeveloperGuide.md

```diff
@@ -1209,20 +1284,28 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
   Use case ends.
 
-* 3a. The given index is invalid.
+* 3a. Nurse's input is invalid command format.
 
     * 3a1. PatientSync shows an error message.
 
       Use case resumes at step 2.
 
-**Use case: Find patients by a tag**
+* 3b. Nurse's input contains invalid patient index.
+
+  * 3b1. PatientSync shows an error message.
+
+    Use case resumes at step 2.
+
+**Use case: UC09 - Delete tag from a patient**
+
+**Preconditions: Tag exists for the target patient**
```

Comment:
better not to use preconditions in use case and put it under extensions

Reply:
Good point! removed the precondition!